package.json

```diff
@@ -16,8 +16,8 @@
     "precompile-svgs": "node utils/precompileSvgs/index.js && yarn run pregenerate",
     "precompile-custom-svgs": "node utils/precompileSvgs/index.js --custom && yarn run pregenerate",
     "_lint-watch-fix": "yarn lint -w -m",
-    "test:percy": "PERCY_LOGLEVEL=error npx percy exec -v -- jest --config jest.conf/visual.index.js -i ./lib/buttons-and-links/__tests__/KButton.spec.js",
-    "test": "jest --config=jest.conf/index.js",
+    "test:percy": "PERCY_LOGLEVEL=info npx percy exec -v -- jest --config jest.conf/visual.index.js -i ./lib/buttons-and-links/__tests__/KButton.spec.js -t '\\[Visual\\]'",
+    "test": "jest --config=jest.conf/index.js -t '^(?!.*\\[Visual\\])'",
```

Comment:
To avoid complex regex patterns, we can also use the `--testPathIgnorePatterns` option, so we just replicate the regex of the previous script just changing the option flag `-t` with `--testPathIgnorePatterns` https://jestjs.io/docs/cli#--testpathignorepatternsregexarray

Reply:
Sure. Will try it out :)

Reply:
Just read that the flag `--testPathIgnorePatterns` is used to ignore specific test files based on their path and not the tests within it. Maybe if we had something like `--testNameIgnorePatterns` it would have been easier.

Reply:
Ooooh it's true, I didn't realize, it seems that jest doesn't have an ignore name pattern. Well, we could keep the regex then

Reply:
Just tested the regex and it seems to be ok :tada: